packages/slate-react/src/utils/set-selection-from-dom.js

```diff
@@ -1,77 +1,14 @@
-import findRange from './find-range'
+import getSelectionFromDOM from './get-selection-from-dom'
```

Comment:
Can you rename this to `setDomSelection`?

Reply:
Please see comment on `getSelectionFromDOM` and let me know if you'd still like to rename it.